php/libraries/Utility.class.inc

```diff
@@ -725,11 +725,12 @@ class Utility
      * This should return an array in a format suitable for addSelect() from
      * NDB_Page
      *
-     * @return associative array of test_names in a Test_Name => "Full Name"
+     * @return array $instruments associative array of test_names in a Test_Name => "Full Name"
```

Comment:
I don't think the variable name is meaningful in the context of defining the return, it's a meaningless internal detail, but other than that +1